File: solutions/typescript/resistor-color-duo/1/resistor-color-duo.ts
export function decodedValue(colors: string[]): number {
  var rValue = 0;
  rValue = (COLORS.indexOf(colors[0]) * 10) + COLORS.indexOf(colors[1]);
  return rValue;
}


export const COLORS = [
  'black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white'
]

File: solutions/python/resistor-color-duo/1/resistor_color_duo.py
COLORS = [
    "black",
    "brown",
    "red",
    "orange",
    "yellow",
    "green",
    "blue",
    "violet",
    "grey",
    "white"
]

def value(colors):
    value = 0
    
    for i in range(0,2):
        value += COLORS.index(colors[i]) * pow(10,2-i-1)
    
    return value


